app/models.py

```diff
@@ -1589,6 +1589,44 @@ def serialize_for_csv(self):
 
         return serialized
 
+    def serialize_permanent_failure(self):
+        return {
+            "billing_code": self.billing_code,
+            "body": None,
+            "completed_at": None,
```

Comment:
I get that we don't send the message, but it seems like completed_at should still be populated?

Reply:
That information is not currently available in request_data and, in my opinion, it would give the wrong impression that something was completed. @k-macmillan , any thoughts on this one?

Reply:
Actually, I see that the field is defined as 
```
def completed_at(self):
        if self.status in NOTIFICATION_STATUS_TYPES_COMPLETED:
            return self.updated_at.strftime(DATETIME_FORMAT)

        return None
```
so it would be wrong to send completed_at if status is not in NOTIFICATION_STATUS_TYPES_COMPLETED ... I'll just call the method and let it decide. @cris-oddball , we'll need to retest then.

Reply:
NOTIFICATION_STATUS_TYPES_COMPLETED  contains NOTIFICATION_FAILED, so it should have a completed_at time, yes?

Reply:
done